spec/unit/space_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'space'
+require 'pg'
```

Comment:
I think because you're already requiring `pg` in `space.rb`, then there's no need to require it here.